notebooks/community/model_garden/model_garden_tfvision_image_object_detection.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<p>redundant imports of os and aiplatform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1651/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>removed</p>